app/assets/javascripts/admin/app/controllers/gobierto_attachments_controller.js

```diff
@@ -431,10 +437,35 @@ this.GobiertoAdmin.GobiertoAttachmentsController = (function() {
             success: function(response, textStatus, jqXHR){
               if(jqXHR.status == 200){
                 Vue.set(self, 'attachments', response.attachments);
+
+                if(response.attachments.length > 0) {
+                  self.showFiles = true;
+                  self.fileListClass = 'fa-caret-down';
+                } else {
+                  self.showFiles = false;
+                  self.fileListClass = '';
+                }
               }
-            },
+            }
           });
         },
+        modifyfileListClass: function(){
+          var self = this;
+
+          if(self.attachments.length > 0) {
+            if(self.showFiles) {
+              self.fileListClass = 'fa-caret-down';
+            } else {
+              self.fileListClass = 'fa-caret-right';
+            }
+          }
+        },
+        toggleList: function(){
+          var self = this;
```

Comment:
In this method you should just do `this.showFiles = !this.showFiles` and don't call the method. The computed will do the magic itself.